tox.ini

```diff
@@ -35,8 +36,6 @@ commands =
     sphinx-build -W -b doctest -d {envtmpdir}/doctrees docs docs/_build/html
     doc8 docs README.rst --ignore-path docs/_build/html
     sphinx-build -W -b linkcheck -d {envtmpdir}/doctrees docs docs/_build/linkcheck
-    python setup.py sdist
-    twine check dist/*
```

Comment:
I'm not sure what the value of this was, because it only printed warnings, rather than failing. I moved it to `release`, as a blocker to `upload`.

Reply:
The README _is_ part of the documentation, even if only on PyPI. Running this here, ensured that it was checked for "correctness" (at least I think that was the justification when it was `readme-renderer` and not `twine check`)

Reply:
Gotcha. Since it's only on PyPI, I still feel okay about it being a failing check in `release`, but I'm open to other suggestions.

Reply:
Actually, I take that back. Since editing the README could be done in a PR, checking its validity should be done during CI. So `twine check --strict` should be part of `docs` or `lint`. I'll make that change..